apps/site/pages/docs/component-customization/override-component.mdx

```diff
@@ -79,14 +85,15 @@ export default overrides
 ```
 
 ### Step 3 - Overriding or Passing additional props to a component
+
 In this step, learn how to override or pass additional props to a component to add a new functionality
 to the component or change its apperance.
 
 For more information on when to override or pass additional prop to a component, and the best practices for overriding components, check the following documentation:
```

Comment:
```suggestion

```

Reply:
Since we do not have these docs, I believe we should delete this section for now.